geoschem.go

```diff
@@ -114,7 +118,15 @@ type GEOSChem struct {
 // 'IJ_AVG_S_xxx'.
 //
 // If msgChan is not nil, status messages will be sent to it.
-func NewGEOSChem(GEOSA1, GEOSA3Cld, GEOSA3Dyn, GEOSI3, GEOSA3MstE, GEOSApBp, GEOSChemOut, VegTypeGlobal, startDate, endDate string, dash bool, msgChan chan string) (*GEOSChem, error) {
+//
+// chemRecordInterval is the time interval between different records in
+// the CTM output, as specified by the user.
+//
+// chemFileInterval is the time interval of each file, as specified by
```

Comment:
... is the time interval of each GEOS-Chem file. ("as specified by the user" can be assumed here)